modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -180,9 +179,11 @@ one IP address pool. If you need to access the services from an external network
 configure load balancers and routers to manage the traffic.
 <10> You must set the platform to `none`. You cannot provide additional platform
 configuration variables for
-ifndef::ibm-z,ibm-z-kvm,ibm-power[your platform.]
-ifdef::ibm-z,ibm-z-kvm[IBM Z infrastructure.]
-ifdef::ibm-power[IBM Power Systems infrastructure.]
+ifndef::ibm-z,ibm-power,rhv[bare metal]
+ifdef::ibm-z[IBM Z]
+ifdef::ibm-power[IBM Power]
+ifdef::rhv[RHV]
+infrastructure.
```

Comment:
This altered the IBM ifdefs. Did you intend to only add in the RHV specific ifdef (and added rhv to the ifndef)?

Reply:
Yes, 
if the product is NOT ibm-z, ibm-power, or rhv, then the term "bare metal" is allowed,
if the product is RHV - then I want the word before "infrastructure." to show up as "RHV"
just as the preview shows it:
"You must set the platform to none. You cannot provide additional platform configuration variables for RHV infrastructure."

Reply:
@emarcusRH great! This looks good for RHV then. The platform agnostic changes that were merged before this is making this a bit complicated. We just need to get this back to how it was previously for the IBM assemblies, too. I think this block should look like this:
```
ifndef::ibm-z,ibm-z-kvm,ibm-power,rhv[your platform.]
ifdef::ibm-z,ibm-z-kvm[IBM Z infrastructure.]
ifdef::ibm-power[IBM Power Systems infrastructure.]
ifdef::rhv[RHV]
```

Reply:
I don't recall is it's me or someone else who made changes to that set of conditions, but the word "infrastucture" is now outside of all these conditions, so the conditions have been changed accordingly.

Reply:
I added a new ifdef line to cover the agnostic case - so that it will say "configuration variables for your infrastructure."

<10> You must set the platform to `none`. You cannot provide additional platform
configuration variables for
ifndef::ibm-z,ibm-power,agnostic, rhv[bare metal]
ifdef::ibm-z[IBM Z]
ifdef::ibm-power[IBM Power]
ifdef::agnostic[your]
ifdef::rhv[RHV]
infrastructure.